app/config.py

```diff
@@ -214,11 +215,11 @@ class Config(object):
                 'schedule': timedelta(minutes=66),
                 'options': {'queue': QueueNames.PERIODIC}
             },
-            'switch-current-sms-provider-on-slow-delivery': {
-                'task': 'switch-current-sms-provider-on-slow-delivery',
-                'schedule': crontab(),  # Every minute
-                'options': {'queue': QueueNames.PERIODIC}
-            },
+            # 'switch-current-sms-provider-on-slow-delivery': {
+            #     'task': 'switch-current-sms-provider-on-slow-delivery',
+            #     'schedule': crontab(),  # Every minute
+            #     'options': {'queue': QueueNames.PERIODIC}
+            # },
```

Comment:
To fully remove the task, I think you'll want to do the same in `app/celery/scheduled_tasks.py`, as well.

Reply:
I decided to just fully remove it, rather than comment out. I don't foresee adding additional SMS providers in the near or medium future

Reply:
Makes sense! I think a next step for future work would be to take the provider priority stuff out of the UI, and then we can return to the API and remove the remaining multi-provider stuff.

Reply:
Yep - agreed. Was mostly planning on tracking that work under #7